Cargo.toml

```diff
@@ -41,7 +41,7 @@ signal-hook = "0.3"
 slab = "0.4"
 socket2 = "0.4"
 thiserror = "1.0"
-time = { version = "0.3.36", features = ["formatting", "local-offset", "macros"] }
+time = { version = "=0.3.36", features = ["formatting", "local-offset", "macros"] }
```

Comment:
Is there a reason to pin the full version?

Reply:
I usually pin to the exact version if the crate is under `1.0.0` to avoid breaking changes

Reply:
Fixed to use `main`'s version.